app/org/sagebionetworks/bridge/config/BridgeConfig.java

```diff
@@ -15,10 +15,6 @@
     private static final String TEMPLATE_CONFIG = CONFIG_FILE;
     private static final String LOCAL_CONFIG = System.getProperty("user.home") + "/" + ".bridge" + "/" + CONFIG_FILE;
 
-    private static final String ENTERPRISE_STORMPATH_ID = "enterprise.stormpath.id";
-    private static final String ENTERPRISE_STORMPATH_SECRET = "enterprise.stormpath.secret";
-    private static final String ENTERPRISE_STORMPATH_APPLICATION_HREF = "enterprise.stormpath.application.href";
```

Comment:
Should these VARs be removed from the bridge-server.conf[1]  and heroku/AWS ENV config?

[1] https://github.com/Sage-Bionetworks/BridgePF/blob/develop/conf/bridge-server.conf#L44